packages/@react-aria/utils/test/useObjectRef.test.js

```diff
@@ -12,8 +12,8 @@
 
 import React, {useEffect} from 'react';
 import {render, renderHook, screen} from '@react-spectrum/test-utils';
-import {useLayoutEffect} from '@react-aria/utils';
-import {useObjectRef} from '../';
+// eslint-disable-next-line rulesdir/useLayoutEffectRule
```

Comment:
lint rule just wasn't quite smart enough to know this is using it from where it requires